aip/general/0151.md

```diff
@@ -59,6 +59,22 @@ rpc CreateBook(CreateBookRequest) returns (google.longrunning.Operation) {
 - APIs with messages that return `Operation` **must** implement the
   [`Operations`][lro] service. Individual APIs **must not** define their own
   interfaces for long-running operations to avoid non-uniformity.
+- If an RPC supports a [validate-only mode](aip-163), the response to a
+  validation request **must** be one of the following:
+  - A successful response with an `Operation` which is already complete, with
+    the `done` field set to `true`, and a valid (but potentially empty) response
+    message in the `result` field, wrapped in a `google.protobuf.Any` message.
+    The `name` field **may** be empty, to avoid the service having to maintain
+    state for successful validation.
```

Comment:
I like being explicit, but do we need to say `done = true` and `result wrapped in an Any`? How else will they pack the `response` field if not via the `Any` that it is typed as?

Reply:
My concern is that readers might set `result = new Any();` - i.e. an entirely empty `Any` wrapper instead of an `Any` wrapper that wraps an empty response message... which I suspect would still confuse client libraries.

Happy to adjust how we word that though.